### PR TITLE
feat: bottender-dialogflow without actions

### PR DESCRIPTION
**Pull request by etrex**
Send fulfillmentMessages to user from Dialogflow when a conversation matched an intent in Dialogflow but can't find the corresponding actions in Bottender.

Only support fulfillmentMessages when the platform equals to 'PLATFORM_UNSPECIFIED' and the message equals to 'text'.

### CODE REVIEW

File: packages/bottender-dialogflow/src/index.ts [hunk 1]
```diff
@@ -54,16 +70,26 @@ module.exports = function dialogflow({
...
       const Action = actions[intent.name] || actions[intent.displayName];
       if (Action) {
-        return withProps(Action as any, { intent, parameters });
+        return withProps(Action as any, { ...queryResult });
+      }
+
+      // fulfillment by Dialogflow
+      const fulfillments = getFulfillments(fulfillmentMessages);
+      if (fulfillments.length > 0) {
+        Promise.all(
```
Comment (chentsulin) [inline packages/bottender-dialogflow/src/index.ts:89]: Why we don't need to wait for those promises get resolved?

File: packages/bottender-dialogflow/src/types.ts [hunk 2]
```diff
@@ -18,8 +18,8 @@ export type QueryResult = {
 
 export type Message = {
   platform: Platform;
-  text: string[];
-};
+  message: string;
```
Comment (chentsulin) [inline packages/bottender-dialogflow/src/types.ts:21]: Is there any reference for this change?

Comment (etrex) [reply packages/bottender-dialogflow/src/types.ts:21]: https://cloud.google.com/dialogflow/docs/reference/rest/v2/projects.agent.intents#Message

Comment (etrex) [reply packages/bottender-dialogflow/src/types.ts:21]: It look likes a undocumented field.

**Pull request merged**